stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/ChatApi.kt

```diff
@@ -65,100 +63,49 @@ internal class ChatApi(
     fun setConnection(userId: String, connectionId: String) {
         this.userId = userId
         this.connectionId = connectionId
+        fileUploader.setConnection(userId, connectionId)
     }
 
     fun sendFile(
         channelType: String,
         channelId: String,
         file: File,
         callback: ProgressCallback
-    ) {
-        val body = ProgressRequestBody(
-            file,
-            callback
-        )
-        val part = createFormData("file", file.name, body)
-
-        retrofitCdnApi
-            .sendFile(
-                channelType,
-                channelId,
-                part,
-                apiKey,
-                userId,
-                connectionId
-            )
-            .call
-            .enqueue(RetroProgressCallback(callback))
-    }
+    ) = fileUploader.sendFile(channelType, channelId, file, callback)
```

Comment:
Explicit return types for these functions would be nice, hard to tell whether they implicitly return something or not